code/game/machinery/cloning.dm

```diff
@@ -126,6 +126,15 @@
 	if (is_operational() && (!isnull(occupant)) && (occupant.stat != DEAD))
 		to_chat(user, "Current clone cycle is [round(get_completion())]% complete.")
 
+/obj/machinery/clonepod/return_air()
+	// We want to simulate the clone not being in contact with
+	// the atmosphere, so we'll put them in a constant pressure
+	// nitrogen. They'll breathe through the chemicals we pump into them.
+	var/global/datum/gas_mixture/immutable/cloner/GM //global so that there's only one instance made for all cloning pods

```

Comment:
No official policy in contributing.md. One keyword is documented in the ref, one is not. We've had the argument before, Remie, and neither one of us ever wins.